flow360/component/flow360_params/flow360_params.py

```diff
@@ -119,6 +119,227 @@ class MeshBoundary(Flow360BaseModel):
+    """
+
+    turbulent_length_scale: Optional[PositiveFloat] = pd.Field(alias="turbulentLengthScale")
+
+
+class ModifiedTurbulentViscosityRatio(Flow360BaseModel):
+    """
+    modifiedTurbulentViscosityRatio : non-dimensional [`-`]
+        The ratio of modified turbulent eddy viscosity (SA) over the freestream viscosity.
+        Applicable only when using SA model.
+    """
+
+    modified_turbulent_viscosity_ratio: Optional[PositiveFloat] = pd.Field(
+        alias="modifiedTurbulentViscosityRatio"
+    )
+
+
+class ModifiedTurbulentViscosity(Flow360BaseModel):
+    """
+    modifiedTurbulentViscosity : non-dimensional [`C_inf*L_gridUnit`]
+        The modified turbulent eddy viscosity (SA). Applicable only when using SA model.
+    """
+
+    modified_turbulent_viscosity: Optional[PositiveFloat] = pd.Field(
+        alias="modifiedTurbulentViscosity"
+    )
+
+
+# pylint: disable=missing-class-docstring
+class SpecificDissipationRateAndTurbulentKineticEnergy(
+    SpecificDissipationRate, TurbulentKineticEnergy
+):
+    pass
+
+
+class TurbulentViscosityRatioAndTurbulentKineticEnergy(
+    TurbulentViscosityRatio, TurbulentKineticEnergy
+):
+    pass
+
+
+class TurbulentLengthScaleAndTurbulentKineticEnergy(TurbulentLengthScale, TurbulentKineticEnergy):
+    pass
+
+
+class TurbulentIntensityAndSpecificDissipationRate(TurbulentIntensity, SpecificDissipationRate):
+    pass
+
+
+class TurbulentIntensityAndTurbulentViscosityRatio(TurbulentIntensity, TurbulentViscosityRatio):
+    pass
+
+
+class TurbulentIntensityAndTurbulentLengthScale(TurbulentIntensity, TurbulentLengthScale):
+    pass
+
+
+class SpecificDissipationRateAndTurbulentViscosityRatio(
+    SpecificDissipationRate, TurbulentViscosityRatio
+):
+    pass
+
+
+class SpecificDissipationRateAndTurbulentLengthScale(SpecificDissipationRate, TurbulentLengthScale):
+    pass
+
+
+class TurbulentViscosityRatioAndTurbulentLengthScale(TurbulentViscosityRatio, TurbulentLengthScale):
+    pass
+
+
+# pylint: enable=missing-class-docstring
+
+TurbulenceQuantitiesType = Union[
+    TurbulentViscosityRatio,
+    ModifiedTurbulentViscosityRatio,
+    ModifiedTurbulentViscosity,
+    SpecificDissipationRateAndTurbulentKineticEnergy,
+    TurbulentViscosityRatioAndTurbulentKineticEnergy,
+    TurbulentLengthScaleAndTurbulentKineticEnergy,
+    TurbulentIntensityAndSpecificDissipationRate,
+    TurbulentIntensityAndTurbulentViscosityRatio,
+    TurbulentIntensityAndTurbulentLengthScale,
+    SpecificDissipationRateAndTurbulentViscosityRatio,
+    SpecificDissipationRateAndTurbulentLengthScale,
+    TurbulentViscosityRatioAndTurbulentLengthScale,
+]
+
+
+# pylint: disable=too-many-arguments, too-many-return-statements, too-many-branches
+def TurbulenceQuantities(
+    viscosityRatio=None,
+    modifiedViscosityRatio=None,
+    modifiedViscosity=None,
+    specificDissipationRate=None,
+    turbulentKineticEnergy=None,
+    turbulentLengthScale=None,
+    turbulentIntensity=None,
```

Comment:
use snake_case for variables. I don't understand why linter does not complain here...
In general, in this python client we follow standard python naming convention:
```python
CONST
def function_name()

class ClassName()

variable_name

```
camelCase is only used as aliases to match solver convention on JSON export.

Reply:
Changed